docs/quick-start.md

```diff
@@ -0,0 +1,17 @@
+# Mirantis Hybrid Cloud Platform
+
+## Installation
+
+### TLDR
+
+> kubectl apply -f https://github.com/Mirantis/hmc/releases/download/v0.0.1/install.yaml
```

Comment:
This looks ok in our README.md, but I believe it's better to have it as a code block on the mkdocs site